mate-panel/panel-toplevel.h

```diff
@@ -138,7 +138,7 @@ PanelState           panel_toplevel_get_state              (PanelToplevel
 
 void                 panel_toplevel_hide                   (PanelToplevel       *toplevel,
 							    gboolean             auto_hide,
-							    GtkDirectionType     direction);
+							    gint                 direction);
```

Comment:
Can you please fix indents? 
I know format issue was still there before.....l

Reply:
@raveit65 done

Reply:
thx.